flutter_news_example/lib/app/view/app.dart

```diff
@@ -65,17 +64,15 @@ class AppView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      themeMode: ThemeMode.light,
-      theme: const AppTheme().themeData,
-      darkTheme: const AppDarkTheme().themeData,
-      localizationsDelegates: AppLocalizations.localizationsDelegates,
-      supportedLocales: AppLocalizations.supportedLocales,
-      home: AuthenticatedUserListener(
-        child: FlowBuilder<AppStatus>(
-          state: context.select((AppBloc bloc) => bloc.state.status),
-          onGeneratePages: onGenerateAppViewPages,
-        ),
+    return RedirectStreamScope(
```

Comment:
can we use refreshListenable instead of this?

Reply:
Yes we can. I'll remove the stream scope and just pass it to the router